AnkiDroid/src/main/java/com/ichi2/anki/snackbar/Snackbars.kt

```diff
@@ -98,7 +109,14 @@ fun Activity.showSnackbar(
     val view: View? = findViewById(R.id.root_layout)
 
     if (view != null) {
-        view.showSnackbar(text, duration, snackbarBuilder)
+        var defaultSnackbarBuilder: SnackbarBuilder? = null
+        if (this is SnackbarConfigurationProvider) {
+            defaultSnackbarBuilder = this.defaultSnackbarBuilder
+        }
```

Comment:
nit: to avoid using a var
```suggestion
        val defaultSnackbarBuilder = if (this is SnackbarConfigurationProvider) {
            this.defaultSnackbarBuilder
        } else {
            null
        }
```

Reply:
Good idea!